### PR TITLE
Sort edges by type

### PR DESCRIPTION
**Pull request by mortid0**
- [x] Forked repo is synced with upstream -> github shows no code delta outside of the desired.
    https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork
- [x] Tests are passing? https://github.com/microsoft/DeepGNN/blob/main/CONTRIBUTING.md#run-tests
- [x] Documentation is added or updated to reflect new code in the same format as the rest of the repo.
- [x] PR is labeled using the label menu on the right side.

Previous Behavior
----------------
Edge index was not sorted by type for each source node. This results in under sampling of neighbors with types smaller than a previous type in index and missing edge features.

New Behavior
----------------
Edges are sorted by type and sampling/extraction will be corrected.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,14 @@
 
 ## 0.1 DeepGNN
 
-### 0.1.52 - 2022-07-27
+### [Unreleased]
```
Comment (coledie) [inline CHANGELOG.md:5]: Remove since unused?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,14 @@
...
+
+### [0.1.53] - 2022-08-02
+
+#### Fixed
+- json/tsv converter didn't sort edges by types resulted in incorrect sampling.
```
Comment (coledie) [inline CHANGELOG.md:10]: This is in a different format than the other section of the change log.

**Review by coledie — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (mortid0) [reply CHANGELOG.md:5]: It seems people are confused where to put unreleased changes based on the existing PRs. Easier to keep this section at all times.

File: CHANGELOG.md (hunk 2 above)
Comment (mortid0) [reply CHANGELOG.md]: Reformatted previous messages.

**Pull request merged**